libs/ngxtension/trackby-id-prop/src/trackby-id-prop.ts

```diff
@@ -0,0 +1,42 @@
+//INSPIRED BY https://medium.com/ngconf/make-trackby-easy-to-use-a3dd5f1f733b
+import { NgForOf } from '@angular/common';
+import {
+	Directive,
+	Input,
+	Provider,
+	inject,
+	type NgIterable,
+} from '@angular/core';
+
+@Directive({
+	selector: '[ngForTrackById]',
+	standalone: true,
+})
+export class NgForTrackByIdDirective<T extends { id: string | number }> {
```

Comment:
nit: what do you think of dropping `Directive` off of the name? Like `NgFor` and `NgIf`, they don't have the `Directive` appended.

Reply:
Totally open to this idea, I've just follow the Angular "best-practices & default" but the name becomes too long, and this is why I decided to export even the TrackByDirectives to "simplify/short" import 😉

What names do you prefer @nartc 
`NgForTrackById`  and `NgForTrackByProp`  
or even shorter:  `TrackById` and `TrackByProp` ?

Reply:
shorter is better